SimG4CMS/Calo/src/CaloSD.cc

```diff
@@ -224,10 +224,9 @@ void CaloSD::EndOfEvent(G4HCofThisEvent* ) {
   
   cleanHitCollection();
   
-#ifdef DebugLog
   edm::LogInfo("CaloSim") << "CaloSD: EndofEvent entered with " << theHC->entries()
```

Comment:
@civanch - it may bring some extra output which was turned off with #ifdef DebugLog